ffwd-client/pom.xml

```diff
@@ -14,6 +14,15 @@
     <description>FastForward client for Java.</description>
 
     <dependencies>
+        <dependency>
+            <groupId>com.google.auto.value</groupId>
+            <artifactId>auto-value</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>javax.annotation</groupId>
```

Comment:
Do we need this one?

Reply:
Yes, library for annotation, it adds annotation to generated code.